index.html

```diff
@@ -512,14 +510,12 @@
         </li>
         <li>Return <var>promise</var> and continue the following steps asynchronously.
         </li>
-        <li>If the <a>webapp</a> is not registered, reject <var>promise</var> with a
-        <code><a>DOMException</a></code> whose name is "<code><a>NotFoundError</a></code>" and
-        terminate these steps.
+        <li>If the <a>webapp</a> is not registered, resolve <var>promise</var> with null.
         </li>
         <li>Retrieve the <a>push registration</a> associated with the <a>webapp</a>.
         </li>
         <li>If there is an error, reject <var>promise</var> with a <code><a>DOMException</a></code>
-        whose name is "<code><a>AbortError</a></code>" and terminate these steps.
+        and terminate these steps.
```

Comment:
What kind of error is an implementation meant to throw if it's not an AbortError? Specifying the kind of DOMException still seems valuable here, even if you keep it as generic as AbortError.